zigzag/zigzag_test_log.py

```diff
@@ -911,6 +937,18 @@ def automation_content(self):
 
         return self._automation_content
 
+    @property
+    def test_execution_parameters(self):
```

Comment:
I think this is the same as in the base class now.  @mwbutcher Can we remove this?